stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_media_attachment.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The naming of this layout is a bit bad because we already have a `stream_ui_media_attachment.xml` layout so I had to add the `message` part to this file.